docs/source/guide/upgrading.rst

```diff
@@ -27,3 +27,11 @@ Notes to refer to when upgrading ``boto3`` versions.
     objects for transfers, use the newly added ``upload_fileobj``
     and ``download_fileobj`` methods that support both nonmultipart and
     multipart transfers.
+
+  * By default, all managed transfer methods are now threaded. In prior
```

Comment:
So I do not think this is wrong. The way I see upgrading notes is that if a customer upgrades to the version listed or any version past that, then the notes should point out at what initial version the noticeable changes started to occur. In the case of this change, these changes occurred at 1.4.0 where threads are **always** used as **may** be used. If I moved this to a 1.4.1 or 1.4.2 section, it would imply that you would not see this change in behavior when moving up to 1.4.0 and only when moving up to 1.4.2 and over, which is not correct. Furthermore, this addition to the upgrading notes is part of a sublist of the overall changes affected in porting the internals of the transfer methods, which makes sense as the change in behavior rooted from 1.4.0 but just did not get a path to remediation till a later version.

I feel that the point that threads are **always** used should be pointed out under the 1.4.0 section because that fact is true. How about if I make that note in the 1.4.0, but also add a 1.4.2 section to help discoverability of the ``use_threads`` note instead of just having it in the final sentence of that bullet point?